Common/src/main/java/at/petrak/hexcasting/api/casting/iota/IotaContainer.java

```diff
@@ -0,0 +1,5 @@
+package at.petrak.hexcasting.api.casting.iota;
+
+public interface IotaContainer {
```

Comment:
This should have a doc comment explaining what the interface is for - basically just what's in the PR description.